prometheus-exporters/esxi-exporter/alerts/service.alerts

```diff
@@ -8,7 +8,7 @@ groups:
       severity: info
       tier: vmware
       service: compute
-      context: "esxi-exporter"
+      context: "esxi-service"
```

Comment:
```suggestion
      context: "esxi-service-critical"
```

To follow our vcsa pattern i would go this route. This is still info, so it doesn't really matter, but we can prepare it already